### DISCUSSION TITLE
Backfill DAG Run stays on queue state when all the tasks succeed

### DISCUSSION THREAD

**Discussion by alejandrofm** [apache/airflow#30930] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

I ran a backfill for a particular task on a DAG and when all the tasks of the DAG run are succeeded, the DAG run stats doesn't change.
Airflow 2.4.2
![image](https://user-images.githubusercontent.com/19591174/234995422-7e0eb51b-c2a7-4362-a23e-6f1f9a7141cb.png)

### What you think should happen instead

The DAG run state should be "successful" when all tasks have been run successfully.

### How to reproduce

Generate a backfill for a particular task of a DAG, run the tasks, the DAG run state doesn't change.

### Operating System

apache/airflow:2.4.2-python3.10

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
There is far too little information to be able to help you to diagnose your problem. The issues reported to Airflow (which are looked at by volunteers that might be able to spare some time to help the people using airflow) need to have enough informatio to be able to do something with it. 

if you would like some advice on the issue -  you need to take a close look. look at the logs and try to describe it in the way (logs, detailed circumstances etc. ) the way you described it is not reproducible without knowing what DAG you have, history of it, details about your deployment and anything els that will be able to help you.

Likely, you have done something (changed the structure of the dag for example) that resulted in the case, but without you explaining it those details, making investigation, looking at logs, and spending some quality time to help the people spending their free time on trying to help you, it's simply not possible to help you.

**Reply by potiuk** [reply]
Converted into discussion - and encourage you to provide more information if you want people here to will to spend some of their free time helping you with your problem.
